lib/puppet/pops/issues.rb

```diff
@@ -894,15 +901,15 @@ def self.hard_issue(issue_code, *args, &block)
   end
 
   EXPRESSION_NOT_SUPPORTED_WHEN_SCRIPTING = issue :EXPRESSION_NOT_SUPPORTED_WHEN_SCRIPTING, :klass do
-    _("%{expr} is only available when compiling a catalog") % { expr: label.a_an_uc(klass) }
+    _("An expression of type %{expression_type} is only available when compiling a catalog.") % { expression_type: label(klass) }
```

Comment:
move "only" to after "available": ...is available only when...

Reply:
updated in latest commit.